content/blog/2025-06-06-a-plan-for-simd.md

```diff
@@ -0,0 +1,142 @@
++++
+title = "A plan for SIMD"
+authors = ["Raph Levien"]
++++
+
+# A plan for SIMD
+
+This document is a followup to the blog post, “[Towards fearless SIMD, 7 years later]”.
+My goal in publishing that was to start a conversation about the best way for Rust to support SIMD programming.
+I’ve also done a bit more exploration and had some discussions at RustWeek, and am now prepared to offer a plan.
+Ideally, I’d like some level of buy-in from the Rust ecosystem, but in any case it feels Good Enough to proceed with writing a large quantity of SIMD code, which is required for Linebender projects to meet their performance goals.
+There's also a companion PR, [fearless_simd#3], which moves this proposal forward.
+
+## Goals
```

Comment:
More than just goals ... who is the intended target user / consumer of this library? Embedded people doing DSP? People shipping end-user applications? Just Linebender? People doing ML inference or training?